fe/fe-core/src/main/java/org/apache/doris/qe/ConnectProcessor.java

```diff
@@ -315,7 +316,8 @@ public void executeQuery(MysqlCommand mysqlCommand, String originStmt) throws Ex
                 || s instanceof UpdateStmt
                 || s instanceof DeleteStmt
                 || s instanceof CreateTableAsSelectStmt
-                || s instanceof CreateTableStmt)) {
+                || s instanceof CreateTableStmt
+                || s instanceof AnalyzeTblStmt)) {
```

Comment:
analyze table do not impl in Nereids, so should not failed even if disabling fallback_to_original_planner

Reply:
Ok. It just makes me confused that `analyze table` can be sucessful but actually it not be Implemented in Nereids.

```
set enable_nereids_planner=true;
set enable_fallback_to_original_planner=false;
```

Reply:
BTW, Should we impl `analyze table` in Neredis side in the future?

Reply:
too many statement has not been impl in Nereids. 
All these statements will fall back to Legacy planner automaticly even if disable `enable_fallback_to_original_planner`

> Should we impl analyze table in Neredis side in the future

Yes, We should impl **ALL** statement in Nereids in feature.

Feel free to impl them in Nereids if you're willing.😁

Reply:
Sure, i will try to impl partial statements in Nereids if i have any bandwidth. 😃 
Thanks.